src/status_im2/contexts/onboarding/sign_in/view.cljs

```diff
@@ -85,14 +85,22 @@
 
 (defn- qr-scan-hole-area
   [qr-view-finder]
-  [rn/view
-   {:style     style/qr-view-finder
-    :on-layout (fn [event]
-                 (let [layout      (js->clj (oops/oget event "nativeEvent.layout")
-                                            :keywordize-keys
-                                            true)
-                       view-finder (assoc layout :height (:width layout))]
-                   (reset! qr-view-finder view-finder)))}])
+  (let [status-bar-height (rn/status-bar-height)]
+    [rn/view
+     {:style     style/qr-view-finder
+      :on-layout (fn [event]
+                   (let [layout      (js->clj (oops/oget event "nativeEvent.layout")
```

Comment:
You may want to use `js->clj` from `utils.transforms`

Reply:
Updated 👍  Thanks, @briansztamfater!